ComPact/Verifier/PactVerifier.cs

```diff
@@ -290,8 +301,38 @@ internal static async Task PublishVerificationResultsAsync(Contract pact, List<T
                 throw new PactException("Publishing verification results failed. Pact Broker returned " + response.StatusCode);
             }
         }
+
+        internal static async Task PublishTags(HttpClient pactBrokerClient, string providerName, string providerVersion, IList<string> tags)
+        {
+            if (string.IsNullOrWhiteSpace(providerName))
+            {
+                throw new PactException("ProviderName should be configured to be able to publish tags.");
+            }
+
+            if (string.IsNullOrWhiteSpace(providerVersion))
+            {
+                throw new PactException("ProviderVersion should be configured to be able to publish tags.");
+            }
+
+            if (tags == null || !tags.Any())
+            {
+                throw new PactException("At least one tag should be configured to be able to publish tags.");
+            }
```

Comment:
I appreciate defensive coding, but none of these can actually occur. ProviderName is taken from the pact itself, so it must be valid, providerVersion is already checked when publishing the verification results, and the tags are checked right before calling this method.